### PR TITLE
Update tensroflow-py

### PR DESCRIPTION
**Pull request by kobrineli**
Build tensorflow with shared libasan and add new python fuzz targets.

### CODE REVIEW

File: projects/tensorflow-py/Dockerfile [hunk 1]
```diff
@@ -32,6 +32,8 @@ RUN python3 -m pip install --upgrade pip
 
 RUN python3 -m pip install --upgrade pyinstaller
 
+RUN pip3 install opencv-python
```
Comment (kobrineli) [inline projects/tensorflow-py/Dockerfile:35]: maybe remove opencv installation

File: projects/tensorflow-py/decode_png.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        pass
+
+
+def main():
+    atheris.Setup(sys.argv, TestOneInput)
+    atheris.Fuzz()
+
+
+if __name__ == "__main__":
+    main()
```
Comment (kobrineli) [inline projects/tensorflow-py/decode_png.py:39]: new line in the end here and in other files

File: projects/tensorflow-py/decode_wav.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
```
Comment (kobrineli) [inline projects/tensorflow-py/decode_wav.py:1]: add license

File: projects/tensorflow-py/encode_wav.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
```
Comment (kobrineli) [inline projects/tensorflow-py/encode_wav.py:1]: add license

File: projects/tensorflow-py/Dockerfile [hunk 5]
```diff
@@ -42,14 +44,32 @@ RUN curl -Lo /usr/bin/bazel \
         && \
     chmod +x /usr/bin/bazel
 
-RUN git clone --depth 1 https://github.com/tensorflow/tensorflow tensorflow
+RUN git clone https://github.com/tensorflow/tensorflow tensorflow && \
+    cd tensorflow && git checkout 6621be46794781c2ecdb0fa1db52b5544ccdfbcb
 
 WORKDIR /tensorflow
```
[the remark this answers is not in the document]
Comment (kobrineli) [reply projects/tensorflow-py/Dockerfile:50]: yes we need

**Review by SweetVishnya — CHANGES_REQUESTED** [review]

File: projects/tensorflow-py/abs.toml [hunk 6]
```diff
@@ -17,5 +17,9 @@
 exit-on-time = 3600
 
 [atheris]
-path = "/tensorflow/tensorflow/tensorflow_src/security/fuzzing/abs_fuzz.py"
+path = "/tensorflow/tensorflow/security/fuzzing/abs_fuzz.py"
 args = "-jobs=1000 -workers=4"
+
+[atheris.env]
+ASAN_OPTIONS = "detect_odr_violation=0"
+
```
[the remark this answers is not in the document]
Comment (kobrineli) [reply projects/tensorflow-py/abs.toml]: done

File: projects/tensorflow-py/abs.toml [hunk 7]
```diff
@@ -17,5 +17,9 @@
 exit-on-time = 3600
 
 [atheris]
-path = "/tensorflow/tensorflow/tensorflow_src/security/fuzzing/abs_fuzz.py"
+path = "/tensorflow/tensorflow/security/fuzzing/abs_fuzz.py"
 args = "-jobs=1000 -workers=4"
+
```
[the remark this answers is not in the document]
Comment (kobrineli) [reply projects/tensorflow-py/abs.toml:22]: done

**Pull request merged**